### ISSUE TITLE
Change Sort order in Post Panel

### ISSUE THREAD

**Issue by Natenom** [razonyang/hugo-theme-bootstrap#756]
### Prerequisites

- [X] I have [searched](https://github.com/razonyang/hugo-theme-bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed feature requests

### Proposal

Currently the comment section is shown under the toc in all cases (when toc is shown in Sidebar and within a blog post) but the icons in the post panel are shown in the reverse order.

### Motivation and context

Don't know what to write here.

**Comment by razonyang** [comment]
Currently, the buttons on post panel are hard-coded, what default order do you prefer? I'll provide a CSS solution to change the default order later.

**Comment by Natenom** [comment]
The order in which the sections appear on the site.

**Comment by razonyang** [comment]
I think it should be fixed by c0b0935, you can also change the order via SCSS:

```scss
.post-panel {
    .action-post-comments {
        order: 4;
    }

    .action-toc {
        order: 5;
    }
}
```

**Comment by Natenom** [comment]
Thank you :)
